src/Suggest/Suggest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchDSL\Suggest;
+
+use ONGR\ElasticsearchDSL\BuilderInterface;
+
+class Suggest implements BuilderInterface
+{
+    const DEFAULT_SIZE = 3;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $text;
+
+    /**
+     * @var array
+     */
+    private $params;
+
+    public function __construct($name, $text, $params = [])
```

Comment:
We should stick to common behaviour and use `ParametersTrait`. You can leave $params in the constructor, but use trait function to set it. Take a look how e.g. https://github.com/ongr-io/ElasticsearchDSL/blob/master/src/Query/GeoShapeQuery.php#L36